### ISSUE TITLE
Releases newer than 3.0.0-beta.1 won't compile or interpret in rollup

### ISSUE THREAD

**Issue by danbulant** [windicss/svelte-windicss-preprocess#69]
My svelte options in rollup look like this:
```
        svelte({
            dev: !production, // run-time checks
            // Extract component CSS — better performance
            css: (css) => css.write(`bundle.css`),
            hot: isNollup,
            preprocess: [
                autoPreprocess({
                    postcss: { plugins: [postcssImport()] },
                    defaults: { style: "postcss" },
                }),
                sveltewindicss.preprocess({
                    // config: "tailwind.config.js",
                    compile: production,          // false: interpretation mode; true: compilation mode
                    // debug: !production,
                    bundle: production,
                    prefix: 'windicss-',       // set compilation mode style prefix
                    globalPreflight: true,  // set preflight style is global or scoped
                    globalUtility: true,    // set utility style is global or scoped
                })
            ],
        }),
```

When I use 3.0.0-beta.1 it works fine as it's supposed to, but after updating to 3.0.0-beta.18, 3.0.0 and 3.0.1 it no longer works (page doesn't include any styles for utility classes).

I didn't try it yet on a small example project but I already tried in two projects with different settings (one without any postcss plugins) and it didn't work in either of them. Using the debug mode would fill my console buffer with just logs, I didn't see any warnings or errors.

**Comment by alexanderniebuhr** [comment]
we deprecated config option with major release so you need to change

```js
bundle: // string to path
globalPreflight: true // remove it is deprecated
globalUtility: true // remove it is deprecated
```

**Comment by danbulant** [comment]
I removed the global properties. How does the `bundle` option work? Is it directly replacement for the `config` option?

On that note, you should update the `README.md` if you deprecated the config option as it's not written there.

I updated to latest public release (3.0.1), I'm still getting a lot of logs with the debug option and styles aren't working correctly.

**Comment by alexanderniebuhr** [comment]
bundle is path to output css file which is written
does not replace config. config is windicss configuration file

```js
bundle: "dist/output.css"
```

the README.md is updated.. if you have any suggestion, a PR is welcome.
Update to 3.0.2... which styles do break??

**Comment by danbulant** [comment]
What should I use for routify? During development, the generated code isn't written to files. I tried using the same value as for svelte css but that didn't work. The earlier versions worked fine without the bundle option.

> does not replace config. config is windicss configuration file

that was just a misunderstanding between us (I thought "we deprecated config option with major release" meant that).

**Comment by danbulant** [comment]
When I set it to the dist folder it fails because during development, that folder doesn't exist

**Comment by alexanderniebuhr** [comment]
i do not know your setup so it is hard to help you. normally you would use a path somewhere in in src for development, and either let your bundler moves it to dist in prod or you have a dynamic path for that.
what does not work with current release? everything should work without bundle option..

i can give you more advice, but i need to either get some more information or be able to reproduce, and currently all our test repos do work without any issue

**Comment by alexanderniebuhr** [comment]
you also might want to come to https://chat.windicss.org for community support

**Comment by danbulant** [comment]
Looks like the issue got fixed by updating to 3.0.2
